compiler/optimizer/InductionVariable.cpp

```diff
@@ -2188,6 +2189,13 @@ bool TR_LoopStrider::examineTreeForInductionVariableUse(TR::Block *loopInvariant
          }
       }
 
+   // Only first-level childerent of array-shadow symbol node have the _arrayShadowParent argument set
+   if (_arrayShadowParent)
+      _arrayShadowParent = false;
+   TR::Node* checkNode = originalNode ? originalNode : node;
+   if (checkNode->getOpCode().hasSymbolReference() && checkNode->getSymbolReference()->getSymbol()->isArrayletShadowSymbol())
```

Comment:
This should be a test on "array shadow symbol" rather than "arraylet shadow symbol" ? The latter won't even be seen in gencon and likely not even in off heap balanced GC, I think